tests/test_basics.py

```diff
@@ -37,6 +38,31 @@ def error_processor(event, exc_info):
     assert event["exception"]["values"][0]["value"] == "aha! whatever"
 
 
+def test_auto_enabling_integrations_does_not_break(sentry_init, caplog):
+    caplog.set_level(logging.DEBUG)
+
+    sentry_init(_experiments={"auto_enabling_integrations": True}, debug=True)
+
+    for import_string in (
+        "sentry_sdk.integrations.django.DjangoIntegration",
+        "sentry_sdk.integrations.flask.FlaskIntegration",
+        "sentry_sdk.integrations.bottle.BottleIntegration",
+        "sentry_sdk.integrations.falcon.FalconIntegration",
+        "sentry_sdk.integrations.sanic.SanicIntegration",
+        "sentry_sdk.integrations.celery.CeleryIntegration",
+        "sentry_sdk.integrations.rq.RqIntegration",
+        "sentry_sdk.integrations.aiohttp.AioHttpIntegration",
+        "sentry_sdk.integrations.tornado.TornadoIntegration",
+        "sentry_sdk.integrations.sqlalchemy.SqlalchemyIntegration",
```

Comment:
Suggestion: extract this sequence such that the tests stay in sync with the implementation.

Reply:
Yeah this makes sense